File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikCoursesAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.courses.StepikCoursesGetQuery;

/**
 * @author meanmail
 */
public class StepikCoursesAction extends StepikAbstractAction {

    public StepikCoursesAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikCoursesGetQuery get() {
        return new StepikCoursesGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikCoursesAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.courses.StepikCoursesGetQuery

class StepikCoursesAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikCoursesGetQuery {
        return StepikCoursesGetQuery(this)
    }
}
